Example/Backpack/View Controllers/GradientSelectorTableViewController.swift

```diff
@@ -22,6 +22,7 @@ class GradientSelectorTableViewController: UITableViewController {
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         guard let destination = segue.destination as? GradientViewController else {
             assert(false, "Destination for gradient segues shoudlbe `GradientViewController`")
+            return
```

Comment:
Building a prod version of the app fails here as the guard statement doesn't actually execution from continuing.